rs/registry/admin/src/main.rs

```diff
@@ -811,6 +814,11 @@ impl ProposalPayload<UpdateIcpXdrConversionRatePayload> for ProposeXdrIcpConvers
 struct StartCanisterCmd {
     #[clap(long)]
     pub canister_id: CanisterId,
+
+    /// If true, the proposal will be sent as an InstallCode proposal instead of ExecuteNnsFunction.
+    /// TODO: Remove `no_` prefix after the InstallCode proposal is supported.
+    #[clap(long)]
+    no_use_legacy_execute_nns_function: bool,
```

Comment:
Note to reviewers: it seems `#[clap(long, default_value = "true")]` doesn't do what I want - even when passing `--use_legacy_execute_nns_function=false` the option evaluates to true.